subsys/bluetooth/host/l2cap.c

```diff
@@ -2392,6 +2506,49 @@ static void l2cap_chan_le_recv_seg(struct bt_l2cap_le_chan *chan,
 	l2cap_chan_le_recv_sdu(chan, buf, seg);
 }
 
+#ifdef CONFIG_BT_L2CAP_SEG_RECV
+static void l2cap_chan_le_recv_seg_direct(struct bt_l2cap_le_chan *chan, struct net_buf *seg)
+{
+	uint16_t seg_offset;
+	uint16_t sdu_remaining;
+
+	if (chan->_sdu_len_done == chan->_sdu_len) {
+
+		/* This is the first PDU in a SDU. */
+
+		if (seg->len < 2) {
```

Comment:
I can't tell if the comment applies to the first `if` statement or the last. Can you make it clearer?